src/background/messages/fetchProjectDigest.ts

```diff
@@ -0,0 +1,42 @@
+import type { PlasmoMessaging } from "@plasmohq/messaging"
+
+import { eventDigestQueryString } from "~queries/eventDigestQuery"
+import { FullProject } from "~types/projectTypes"
+import { UserSession } from "~types/userTypes"
+import determineActiveProjectId from "~utils/determineActiveProjectId"
+import { fetchStrapiContent } from "~utils/fetchStrapiContent"
+import generatePagination from "~utils/generatePagination"
+import newStorage from "~utils/newStorage"
+
+const handler: PlasmoMessaging.MessageHandler = async (req, res) => {
+  const storage = newStorage()
+  const userSession: UserSession = await storage.get(
+    "arebyte-audience-session"
+  )
+  const digestDayCount: number = await storage.get(
+    "arebyte-digest-count"
+  )
+
+  const activeProjectId = await determineActiveProjectId(
```

Comment:
I think I'm a little confused about the need for this db call. Won't this just be the current project I'd in the user context store?

Reply:
yep - now getting this data directly from the cms